docs/concepts/models/overview.md

```diff
@@ -395,6 +395,47 @@ Some properties are only available in specific model kinds - see the [model conf
 ### disable_restatement
 :   Set this to true to indicate that [data restatement](../plans.md#restatement-plans) is disabled for this model.
 
+### auto_restatement_cron
+:   A cron expression that determines when SQLMesh should automatically restate this model. Restatement means re-evaluating either a number of last intervals (controlled by [`auto_restatement_intervals`](#auto_restatement_intervals)) for model kinds that support it or the entire model for model kinds that don't. Downstream models that depend on this model will also be restated. The auto-restatement is only applied when running the `sqlmesh run` command against the production environment.
+
+    A use case for auto-restatement is to re-evaluate a model periodically (less frequently than the model's cron) to account for late-arriving data and / or dimension changes. Instead of setting the [`lookback`](#lookback) property, which applies during each model evaluation, users can leverage auto-restatements to process all late-arriving data in one go, frontloading the cost.
+
+    Unlike the [`lookback`](#lookback) property, which only controls the time range of data scanned, auto-restatement rewrites all previously processed data for this model in the target table.
```

Comment:
we should strongly discourage people from using this as it has negative tradeoffs

Reply:
Would it be fair to say, "Use this as a last resort. If you're on the fence using this feature, please reach out on our slack community https://tobikodata.com/slack"?

Reply:
How do you suggest expressing this?

Reply:
Using this indicates a problem with data modeling. You should not being doing joins early in your pipelines because dimensions change, keep things normalized until as late as possible.

In terms of late arriving data, you should use lookback.

When you opt into this feature, you lose deployability, meaning no promotion from dev to prod. This is because the data is no longer consistent.

Reply:
reworded